controllers/authzctrl/suite_test.go

```diff
@@ -43,9 +43,11 @@ var _ = SynchronizedBeforeSuite(func(ctx context.Context) {
 						},
 						Resources: "components",
 					},
-					WorkloadSelector: map[string]string{},
-					Ports:            []string{},
-					HostPaths:        []string{"spec.host"},
+					WorkloadSelector: map[string]string{
```

Comment:
It would be good from the consumer point of view to extend godoc for this struct similarly to `RoutingTarget`.

Reply:
That was indeed my intention. late night hacking. hehe

https://github.com/maistra/odh-platform/pull/62/commits/a0bc1616e75c50db5b09a37e40be40f030c76a1d